yorkie/metrics/metrics_server.go

```diff
@@ -0,0 +1,58 @@
+package metrics
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
I think `go.mod` was updated when `prometheus` modules were added. Should we add them together to this commit?